x-pack/test/api_integration/apis/metrics_ui/inventory_threshold_alert.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { convertToKibanaClient } from '@kbn/test';
+import {
+  Comparator,
+  InventoryMetricConditions,
+} from '../../../../plugins/infra/server/lib/alerting/inventory_metric_threshold/types';
+import { InfraSource } from '../../../../plugins/infra/server/lib/sources';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { DATES } from './constants';
+import { evaluateCondition } from '../../../../plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition';
+import { InventoryItemType } from '../../../../plugins/infra/common/inventory_models/types';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const esClient = convertToKibanaClient(getService('es'));
+
+  const baseCondition: InventoryMetricConditions = {
+    metric: 'cpu',
+    timeSize: 1,
+    timeUnit: 'm',
+    sourceId: 'default',
+    threshold: [100],
+    comparator: Comparator.GT,
+  };
+
+  const source: InfraSource = {
+    id: 'default',
+    origin: 'internal',
+    configuration: {
+      name: 'Default',
+      description: '',
+      logIndices: {
+        type: 'index_pattern',
+        indexPatternId: 'some-test-id',
+      },
+      metricAlias: 'metricbeat-*',
+      inventoryDefaultView: 'default',
+      metricsExplorerDefaultView: 'default',
+      anomalyThreshold: 70,
+      fields: {
+        message: ['message'],
+      },
+      logColumns: [
+        {
+          timestampColumn: {
+            id: '5e7f964a-be8a-40d8-88d2-fbcfbdca0e2f',
+          },
+        },
+        {
+          fieldColumn: {
+            id: ' eb9777a8-fcd3-420e-ba7d-172fff6da7a2',
+            field: 'event.dataset',
+          },
+        },
+        {
+          messageColumn: {
+            id: 'b645d6da-824b-4723-9a2a-e8cece1645c0',
+          },
+        },
+      ],
+    },
+  };
+
+  const baseOptions = {
+    condition: baseCondition,
+    nodeType: 'host' as InventoryItemType,
+    source,
+    logQueryFields: void 0,
+    esClient,
+    compositeSize: 10000,
+    startTime: DATES['8.0.0'].hosts_only.max,
+  };
+
+  describe('Inventory Threshold Rule Executor', () => {
+    before(() => esArchiver.load('x-pack/test/functional/es_archives/infra/8.0.0/hosts_only'));
+    after(() => esArchiver.unload('x-pack/test/functional/es_archives/infra/8.0.0/hosts_only'));
```

Comment:
You seem to be missing an `await` here
```suggestion
    before(async () => await esArchiver.load('x-pack/test/functional/es_archives/infra/8.0.0/hosts_only'));
    after(async () => await esArchiver.unload('x-pack/test/functional/es_archives/infra/8.0.0/hosts_only'));
```

Reply:
That's weird, all the other tests don't require an async and await for the same expression. It also runs fine without that locally. I suspect there was a hiccup in the test framework. Let's see if it resolves itself.

Reply:
It looks like the problem was with the `convertToKibanaClient`being called outside the test. It appears to only recast the type for the client so I'm not sure why that's blowing stuff up in CI but not locally. 🤷